backend/routes/db.py

```diff
@@ -725,23 +726,100 @@ def next_api_call_group_id() -> int:
     return id
 
 
-def usage_query():
+def usage_query(verbose=True) -> List[Dict]:
     """Query for usage data"""
+    t0 = datetime.now()
     with get_db_connection() as con:
-        data = sql_query(con, """
+        # what we had originally
+        # data: List[RowMapping] = sql_query(con, """
+        #     SELECT DISTINCT r.*, array_sort(g.api_calls) api_calls, g.duration_seconds, g.group_start_time
+        #     FROM public.apiruns_grouped g
+        #     RIGHT JOIN public.api_runs r ON g.api_call_group_id = r.api_call_group_id""")  # 530,232
+        data: List[RowMapping] = sql_query(con, """
             SELECT DISTINCT r.*, array_sort(g.api_calls) api_calls, g.duration_seconds, g.group_start_time
-            FROM public.apiruns_grouped g
-            RIGHT JOIN public.api_runs r ON g.api_call_group_id = r.api_call_group_id""")
+            FROM public.api_runs r
+            LEFT JOIN public.apiruns_grouped g ON g.api_call_group_id = r.api_call_group_id
+            WHERE g.api_call_group_id != -1 AND g.api_call_group_id IS NOT NULL;""")  # 13,210
```

Comment:
I definitely want to understand why/if that's the case. The only reason I can think of is that left/inner will effectively be the same now because now we expect that the distinct values for `api_call_group_id` are the same in both tables. But that seems like what we want.

Certainly for some of the statistics, I'll want to get rid of these NULL and -1 groups. But perhaps for others, I think now that you might want me to sample from all of the `api_runs`, regardless, possibly. E.g. total number of errors. But any cases where I do that, now our stats will be proportionally out of sync.